concepts/Common/Licensing/00 Licensing.md

```diff
@@ -0,0 +1,3 @@
+A license is required to use DevExtreme components, you can purchase it on the [pricing page](https://js.devexpress.com/Buy/). The license includes 12 months of access to professional support and updates. Refer to the following page to learn more: [Licensing](https://js.devexpress.com/Licensing/).
+
+Starting from the v23.2, DevExtreme checks the license; if you do not have a license, an error message will be displayed. To ensure successful completion of the license verification procedure, you must supply a license key.
```

Comment:
JFYI: don't forget to update this line if we add license check to older versions (v22.2)

Reply:
How can we learn about it? In case we did it, can you maybe drop a card to the tw board that we should update this info?